fundingmanager.go

```diff
@@ -3192,6 +3186,25 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 	commitType := commitmentType(
 		msg.peer.LocalFeatures(), msg.peer.RemoteFeatures(),
 	)
+
+	// First, we'll query the fee estimator for a fee that should get the
+	// commitment transaction confirmed by the next few blocks (conf target
+	// of 3). We target the near blocks here to ensure that we'll be able
+	// to execute a timely unilateral channel closure if needed.
+	commitFeePerKw, err := f.cfg.FeeEstimator.EstimateFeePerKW(3)
```

Comment:
While we're here, maybe also reduce this to 6-18 confs? As lately the 3 block estimate can be rather high

Reply:
You mean for all channel types, or only for anchors?

If it is for the general case, I would prefer to do it in a separate PR. Maybe even make it a config option?